docs/administration-guide/gaffer-stores/federated-store.md

```diff
@@ -2,4 +2,405 @@
+[
+  "GraphA", "GraphB"
+]
+```
+
+## Adding Graphs
+
+To add a graph to a Federated Store you use the `AddGraph` operation. To load a graph you need to provide:
+
+- Graph ID
+- Graph Schema
+- Graph Properties
+
+!!! note "Duplicate Graph IDs"
+    You can't add a graph with an ID which is already in use, you will need to explicitly remove the old graph first.
+
+A Schema & Properties are not required if `parentSchemaId`/`parentPropertiesId` are instead used to retrieve this information from a Graph Library.
+
+You can limit user access to when adding sub-graphs, see [Limiting Access](#limit-access-with-authentication).
+User specified store properties are allowed by default, but this can be disallowed if desired, see [limit custom properties](#limit-custom-properties).
+
+```json title="AddGraph Operation"
+{
+  "class" : "AddGraph",
+  "graphId" : "AnotherGraph",
+  "schema" : {
+    ... Schema in JSON format
+  },
+  "storeProperties" : {
+    ... Store Properties
+  }
+}
+```
+
+??? example "AddGraph with the Java API"
+
+    ```java
+    AddGraph addAnotherGraph = new AddGraph.Builder()
+            .graphId("AnotherGraph")
+            .schema(mySchema)
+            .storeProperties(myStoreProperties)
+            .build();
+    federatedGraph.execute(addAnotherGraph, user);
+    ```
+
+## Removing Graphs
+
+To remove a graph from a Federated Store, you only need to know the graph's ID. Removing a graph does not delete it, it only removes it from the scope of the store.
+A removed graph can be re-added again later, assuming the same graph ID is used and the configuration for the underlying store and any associated storage is unchanged.
+Graphs can only be removed by users with write access or by an administrator (user with admin auths).
+
+```json title="RemoveGraph Operation"
+{
+  "class" : "RemoveGraph",
+  "graphId" : "AnotherGraph"
+}
+```
+
+??? example "RemoveGraph with the Java API"
+
+    ```java
+    RemoveGraph removeGraph = new RemoveGraph.Builder()
+            .graphId("AnotherGraph")
+            .build();
+    federatedGraph.execute(removeGraph, user);
+    ```
+
+## Removing Graphs and Deleting Data
+
+Remove a graph and deleting the underlying data is done with the `RemoveGraphAndDeleteAllData` operation.
+Other than also deleting data, this operation is identical to `RemoveGraph`.
+Gaffer stores are not required to support this operation, if it's not supported an error will be raised.
+It is supported by and has been tested with the Accumulo Store.
+
+```json title="RemoveGraphAndDeleteAllData Operation"
+{
+  "class" : "RemoveGraphAndDeleteAllData",
+  "graphId" : "AnotherGraph"
+}
+```
+
+??? example "RemoveGraphAndDeleteAllData with the Java API"
+
+    ```java
+    RemoveGraph removeGraphAndDeleteAllData = new RemoveGraphAndDeleteAllData.Builder()
+            .graphId("AnotherGraph")
+            .build();
+    federatedGraph.execute(removeGraphAndDeleteAllData, user);
+    ```
```

Comment:
You mean the little tabs with "JSON, Python, Java"? I agree that would be an improvement, it would need the relevant request in Python as well.

I think this could be done at the same time as moving these operations into the operations guide.

Reply:
Yes, sounds good to me.